code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -438,6 +443,18 @@
 	throw_speed = 1
 	throwforce = 0
 
+/obj/item/reagent_containers/food/snacks/lollipop/equipped(mob/user, slot)
+	. = ..()
+	chewing = (chewing-1)*-1
```

Comment:
What the
```suggestion
	chewing = !chewing
```

Reply:
:trollface:

Reply:
Wait this doesnt make any sense, for a start it doesnt check the equipped slot so it will be chewed in hand and why would you toggle chewing on/off when its equipped to mouth.
It should set chewing to true when equipped to the mask slot, then set chewing to false if the chew time fails (allowing the timer to be restarted).

Reply:
I think it did this at some point, I might've just forgot to add it back. 
Whoops.